classes/submission/sectionEditor/SectionEditorSubmissionDAO.inc.php

```diff
@@ -1305,6 +1305,61 @@ function getProofreaderStatistics($journalId, $proofreaderId = null) {
 
 		return $statistics;
 	}
+	
+	/* Draft editorial https://github.com/ubiquitypress/OJS-Draft-Editorial/commit/9135893848e9d40923c105a2cafc59b0529659c9 */
+	function recordDraftDecision($articleID, $Key, $seniorEditorID, $juniorEditorID, $Subject, $Body, $Note, $Decision, $Status) {
+		$this->update(
+			'INSERT INTO draft_decisions
+				(key_val, senior_editor_id, junior_editor_id, article_id, decision, subject, body, note, status)
+				VALUES
+				(?, ?, ?, ?, ?, ?, ?, ?, ?)',
+			array($Key, $seniorEditorID, $juniorEditorID, $articleID, $Decision, $Subject, $Body, $Note, $Status)
+		);
+	}
+
+	function getArticleDrafts($articleId) {
+		$result =& $this->retrieve(
+			'SELECT dd.key_val, dd.decision, dd.article_id, u.first_name, u.last_name
+			 FROM draft_decisions dd
+			 JOIN users u ON dd.junior_editor_id = u.user_id
+			 WHERE article_id = ? AND dd.status = ?',
+			array($articleId, 'draft')
```

Comment:
When possible, cast variables explicitly in DB calls. `$articleId` should be `(int) $articleId`